lib/stoplight/light/config.rb

```diff
@@ -69,6 +69,27 @@ def ==(other)
         other.is_a?(self.class) && to_h == other.to_h
       end
 
+      # @param error [Exception]
+      # @return [Boolean]
+      def track_error?(error)
+        skip = skipped_errors.any? { |klass| klass === error }
+        track = tracked_errors.any? { |klass| klass === error }
+
+        !skip && track
+      end
+
+      # @param number_of_errors [Numeric]
+      # @return [Boolean]
+      def threshold_exceeded?(number_of_errors)
+        number_of_errors == threshold
```

Comment:
```suggestion
        number_of_errors >= threshold
```
I'm not sure if we can arrive at this state (like if we fallback to memory and then go back to redis and find out that number of errors exceed the threshold), but I think we will be safer this way 🙂